Identity.sln

```diff
@@ -86,6 +87,10 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "IdentitySample.DefaultUI",
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.AspNetCore.Identity.UI", "src\UI\Microsoft.AspNetCore.Identity.UI.csproj", "{894E102D-56D4-4B02-8F13-8781F4324C3E}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Identity.DefaultUI.WebSite", "test\WebSites\Identity.DefaultUI.WebSite\Identity.DefaultUI.WebSite.csproj", "{CAE02AD2-F941-4ACB-B469-13EFF551BB74}"
```

Comment:
Should we just use the ui sample for the functional test?  kills two birds with one stone as we don't have to worry about the sample not working :)

Reply:
Oh never mind, the test website is doing most of the test/asserts

Reply:
As we discussed, I think long term it would be ideal if we could point the functional tests against the sample ui site, maybe something to track as a future enhancement/issue